lawlibrary/templates/peachjam/gazette_detail.html

```diff
@@ -6,7 +6,7 @@
     <ol class="breadcrumb">
```

Comment:
this file is now identical to peachjam's gazette_detail so the lawlibrary one can fall away